src/logger.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+"""Logger module"""
+
+import logging
+import logging.config
+
+
+class Logger:
+    """Logging utility class
+
+    This class accepts `name` of the logger in its contructor, by default
+    `root` will be used if not provided.
+    """
+
+    def __init__(self, name="root"):
+        self.logger = self.get_logger(name)
+
+    def get_logger(self, name):
+        """Returns logger object using configurations and logger name"""
+        logging.config.fileConfig('/home/kernelci/config/logger.conf')
```

Comment:
This hard-coded path to the config file is likely to break at some point.  It means the code can only work inside the Docker image that sets up a `kernelci` user etc.  I think it would be better to have the config file name as an argument passed to `__init__()`.  Also this method could be renamed to `_get_logger()` as it's not really a public one but something used by the constructor to create the internal `self._logger` object.